docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -132,5 +137,4 @@ ${BASH_ALIASES[openstack]} server list | grep -F '| test | ACTIVE |' && \
 ----
 
 [NOTE]
-After the data plane adoption, the hosts continue to run Red Hat Enterprise Linux (RHEL)
-{rhel_prev_ver}. To take advantage of RHEL {rhel_curr_ver}, perform a minor update procedure after finishing the adoption procedure.
+After the data plane adoption, the Compute hosts continue to run Red Hat Enterprise Linux (RHEL) {rhel_prev_ver}. To take advantage of RHEL {rhel_curr_ver}, perform a minor update procedure after finishing the adoption procedure.
```

Comment:
Is the " minor update procedure" updating the RHEL version?
Maybe could specifically say to update the RHEL version, or does the  minor update procedure do all that for you?

Reply:
I think this may be a minor RHEL update, yes.

@SeanMooney